pkg/kubelet/server/server.go

```diff
@@ -706,6 +779,11 @@ func (s *Server) getRun(request *restful.Request, response *restful.Response) {
 	writeJsonResponse(response, data)
 }
 
+func (s *Server) debuggingDisabled(request *restful.Request, response *restful.Response) {
+	message := []byte("Server endpoints for log collection and local running of containers and commands are disabled.")
+	writeJsonResponse(response, message)
```

Comment:
What's the response status code after this change?
404 or something?

Reply:
Seems like it will be 200 (statusOK).

Reply:
And I think we should pass a right `json` to function write`Json`Response

Reply:
Maybe:
```
func (s *Server) debuggingDisabled(response *restful.Response) {
	message := "Server endpoints for log collection and local running of containers and commands are disabled."
	response.WriteError(http.StatusMethodNotAllowed, fmt.Errorf(message))   // not sure 405 is ok here, or 404?
}

```

Reply:
I think 405 is the right code here.